tests/unit/BaseTestCase.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/*
+ * This class caters for the different PHPUnit versions between production 
+ * and development.
+ * To execute unit tests especially during development set an environment
+ * variable TEST_ENV=<environment> either as 'dev' or 'prod' :
+ * 
+ *   export TEST_ENV=<environment>
+ *   php codecept.phar run unit --debug
+ */
+$test_env = null !== getenv('TEST_ENV') ? getenv('TEST_ENV') : 'prod';
```

Comment:
`$test_env` is probably not a good attribute name, especially if it can have the `prod` value. What about just `$env`?

Reply:
Also, `prod` is the CI environment?

Reply:
Actually 'prod' is just a placeholder that stands for: 'NOT development environment'. AFAIK the CI environment does not have a distinction of environment.